app/uk/gov/hmrc/lisaapi/config/AppContext.scala

```diff
@@ -40,4 +39,10 @@ class AppContext @Inject()(
   lazy val v2endpointsEnabled = runModeConfiguration.getBoolean("api.endpointsEnabledv2").getOrElse(throw new RuntimeException(s"Missing key api.endpointsEnabledv2"))
 
   override protected def mode = environment.mode
+
+  def endpointIsEnabled(endpoint: String): Boolean = {
+    val endpointIsDisabled = runModeConfiguration.getStringList("api.disabledEndpoints").fold(false)(list => list.contains(endpoint))
```

Comment:
Should the default be true or false?

Reply:
I think the default state should be that every API is available unless they're explicitly disabled. Then if we have every API available there's no config required at all.

Reply:
Could we use notContains? Or have the method as endpointIsDisabled?

Reply:
Hahaha, already done so!